kpi/serializers/v2/service_usage.py

```diff
@@ -1,22 +1,37 @@
+from django.db.models import Sum, Q
+from django.db.models.functions import Coalesce
+from django.utils import timezone
+from djstripe.models import Subscription
```

Comment:
This import appears to generate a RuntimeError at startup when stripe isn't configured. We decided that stripe should be optional. I suspect you could do a condition check and run organizations.something to fetch check if there is a subscription without needing to import.